src/demangle.cc

```diff
@@ -35,10 +35,16 @@
 // Note that we only have partial C++0x support yet.
 
 #include <stdio.h>  // for NULL
+#include "utilities.h"
 #include "demangle.h"
 
+#if defined(OS_WINDOWS)
+#include <Dbghelp.h>
```

Comment:
Got it. This is where I'm going to add the `pragma` for automatic linking as well.